### PR TITLE
Select user component

### PR DESCRIPTION
**Pull request by luizbraga**
**Link to task:**
[Trello](https://trello.com/c/wwh5Bgiw)
[Zeplin](https://app.zeplin.io/project/5a2044f77156a8d3835aeb1e/screen/5bb67c7eb66b4c436f00a62e)

**How to test:**
Run `npm run dev` and access the component in `http://localhost:9001/`, then go to `Select user > complete version`. It should look like the one in Zeplin.

The horizontal limit is ~= 350px

**Screenshot**
[Small demo the component functionality](https://monosnap.com/file/62ZjMJEmH9c0iZqHtaGVH92DUKi7oa)

### CODE REVIEW

File: src/components/select-user/select-user.stories.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { withKnobs, boolean, text } from '@storybook/addon-knobs/react';
+
+import SelectUser from './index';
+
+const stories = storiesOf('Select User', module);
+stories.addDecorator(withKnobs);
+
+const userObj = {
+    id: 1,
+    userName: 'King of cacimbinha',
```
Comment (thulioph) [inline src/components/select-user/select-user.stories.js:13]: The best name 🤣🤣

File: src/components/select-user/index.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        <UserImage>
+            <UserPhoto>
+                <img src={photoUrl} alt={userName} />
+            </UserPhoto>
+        </UserImage>
+        <UserName>
+            {userName}
+        </UserName>
+        <SelectButton>
+            {isSelected ? <CloseIcon scale="0.5" strokeWidth="1px" /> : <CloseIcon fill="#00a7ac" rotate="45deg" scale="0.4" strokeWidth="1px" />}
```
Comment (thulioph) [inline src/components/select-user/index.jsx]: Hey @luizbraga what do you think about this kind of indentation?

```suggestion
            {isSelected 
              ? <CloseIcon scale="0.5" strokeWidth="1px" /> 
              : <CloseIcon fill="#00a7ac" rotate="45deg" scale="0.4" strokeWidth="1px" />
            }
```

File: src/components/select-user/index.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    handleClick: PropTypes.func,
+    userId: PropTypes.string,
+    userName: PropTypes.string,
+    photoUrl: PropTypes.string,
+  };
+  
+  SelectUser.defaultProps = {
+    isSelected: false,
+    handleClick: null,
+    userId: '0',
```
Comment (thulioph) [inline src/components/select-user/index.jsx]: I'm not sure about the default value of the `userId` with a value different of an empty string.

File: src/components/select-user/index.jsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    img {
+        max-width: 100%;
+    }
+`;
+
+const UserName = styled.div`
+    flex: 1 1 30%;
+    padding-top: 7px;
+    padding-left: 16px;
+    padding-bottom: 6px;
```
Comment (thulioph) [inline src/components/select-user/index.jsx]: Two notes about it:

### too close values
In cases were you have too close values as `7px` and `6px`, I prefer to use `6px` to keep the consistency.

In this case:

```suggestion
    padding: 6px 16px;
```

### shorthand
When we have more than one value for the same property, is recomended to use the [shorthand](https://developer.mozilla.org/pt-BR/docs/Web/CSS/padding) version. 

```js
// padding: top right left bottom; (structure of the values)
// padding: 6px (top and bottom) 16px (right and left)
```

File: src/components/select-user/index.jsx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    text-align: left;
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
```
Comment (thulioph) [inline src/components/select-user/index.jsx]: Values like `normal`, we normally don't use, they will be the default of the user.

File: src/components/select-user/index.jsx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    letter-spacing: -0.3px;
+    color: #4a4a4a;
+`;
+
+const SelectButton = styled.div`
+    flex: 0;
+    text-align: right;
+    margin: 0;
+    padding-top: 8px;
+    padding-right: 8px;
```
Comment (thulioph) [inline src/components/select-user/index.jsx]: The same as mentioned above.

File: src/components/select-user/index.jsx (hunk 4 above)
Comment (luizbraga) [reply src/components/select-user/index.jsx]: Thanks for the tip!

File: src/components/select-user/index.jsx (hunk 3 above)
Comment (luizbraga) [reply src/components/select-user/index.jsx]: Makes sense, thanks!

**Review by thulioph — APPROVED** [review]

**Pull request merged**